content/source/docs/enterprise/before-installing/index.html.md

```diff
@@ -128,29 +126,64 @@ See [Network Requirements](./network-requirements.html) for details.
 
 <a id="software-requirements"></a>
 
-### Software Requirements (Standalone Deployment)
-
-Every Terraform Enterprise installation requires the following:
-Enterprise:
-
-- A 64-bit architecture Linux-based operating system
-- Linux Kernel 3.10 or greater
-- Docker Engine 17.06.2-ce to 20.10:
-  - Docker Engine 18.01.0-ce or greater requires Replicated 2.32.0
-    or greater
-  - Support for [Alpine 3.14](https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.14.0#faccessat2)
-    Docker images:
-    - Docker Engine 17.06.2-ce to 20.09 requires runc v1.0.0-rc93
-    - Docker Engine 20.10 or greater requires libseccomp 2.4.4
-  - In Online mode, the installer will install Docker Engine
-    automatically
-  - In Airgapped mode, you must install Docker Engine before running
-    the installer
-
-Some operating systems also have additional requirements:
-
-- [RedHat Enterprise Linux (RHEL) Requirements](./rhel-requirements.html)
-- [CentOS Requirements](./centos-requirements.html)
+### Docker Engine Requirements
```

Comment:
Do we want to cover the 3rd case of overriding the default `libseccomp` profile?

https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.14.0#faccessat2

Reply:
I think it's probably reasonably to include for customers who can't upgrade Docker immediately.

Reply:
Done!